docs/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NASA201 Gerald Louie Moral</title>
+</head>
+<body>
+    <head>
+        <h1>        Louie's Website</h1>
+    </head>
+    <main>
+        <h2> The First Lab for Nasa201</h2>
+        <p> Cats and why they are one of the being loved animals in the world</p>
+        <img src="image/cat_image.jpg" alt="catimage.com" width="250" height="250">
+        <img src="image/Cat NASA201.jpg" alt="catimage2.com" width="250" height="250">
+        <img src="image/Cat NASA201.12.jpg" alt="catimage3.com" width="250" height="250">
+        <img src="image/Cat NASA201.13.jpg" alt="catimage3.com" width="250" height="250">
+
+        
+    </main>
+<h2>Introduction</h2>
+<p>
+    Cats: Feline Majesty in Domestic Grace
+    
+    Cats, with their enigmatic allure and independent spirits, have long captivated the hearts of humans. Revered in ancient civilizations and cherished in contemporary homes, these graceful creatures embody a unique blend of elegance and mystique.
+    </p>
+    <h2> Let's Explore the Cat Kingdom</h2>
+    <p>One of the most striking features of cats is their innate sense of autonomy. Unlike many domesticated animals, cats maintain a distinct aura of independence, navigating their surroundings with an air of self-assurance. Their aloof demeanor often belies a deep-rooted intelligence, as evidenced by their ability to problem-solve and adapt to diverse environments.
+
+        Beyond their independent nature, cats possess an unparalleled grace that is both captivating and mesmerizing. From the fluidity of their movements to the sleek contours of their bodies, every aspect of a cat's form exudes an innate sense of beauty. Whether lounging in the sun or prowling through moonlit shadows, cats move with a sense of purpose that commands admiration and respect.</p>
+
+        
+          
+          
+        <!DOCTYPE html>
+<html>
```

Comment:
This is a new HTML document.  Everything from here to the first closing </html> should be in a separate file